src/main/scala/edg_ide/runner/DesignTopRunConfigurationType.scala

```diff
@@ -80,37 +92,66 @@ class DesignTopRunConfiguration(project: Project, factory: ConfigurationFactory,
 
   val kFieldDesignName = "DESIGN_NAME"
   val kFieldNetlistName = "NETLIST_NAME"
+  val kFieldToggle = "TOGGLE_NAME"
```

Comment:
Similarly, the naming and value can be more descriptive - but consistent with the Enum name